common/main/bs_Cyrl.xml

```diff
@@ -8319,351 +8319,351 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			</unit>
 			<unit type="duration-minute">
 				<displayName>минут</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
+				<unitPattern count="one">{0} мин.</unitPattern>
+				<unitPattern count="few">{0} мин.</unitPattern>
 				<unitPattern count="other">{0} мин.</unitPattern>
 				<perUnitPattern>{0} у мин.</perUnitPattern>
 			</unit>
 			<unit type="duration-second">
 				<displayName>секунде</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
+				<unitPattern count="one">{0} сек.</unitPattern>
+				<unitPattern count="few">{0} сек.</unitPattern>
 				<unitPattern count="other">{0} сек.</unitPattern>
 				<perUnitPattern>{0} у сек.</perUnitPattern>
 			</unit>
 			<unit type="duration-millisecond">
 				<displayName>мс</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
+				<unitPattern count="one">{0} мс</unitPattern>
+				<unitPattern count="few">{0} мс</unitPattern>
 				<unitPattern count="other">{0} мс</unitPattern>
 			</unit>
 			<unit type="duration-microsecond">
 				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
+				<unitPattern count="one">{0} μs</unitPattern>
+				<unitPattern count="few">{0} μs</unitPattern>
 				<unitPattern count="other">↑↑↑</unitPattern>
```

Comment:
This is where we get the first "incomplete logical group" error for bs_Cyrl.

Currently the main branch has ↑↑↑ for all three paths in the group (one/few/other). The "hardening" with -fV affected two of those paths (one/few). The third one (other) gets the error. The inheritance is from root. The method LogicalGroupChecker.isHereOrNonRoot returns false.

See https://st.unicode.org/cldr-apps/v#/bs_Cyrl/Duration/691eb9dd1fae68d3 (bs_Cyrl, narrow-other-nominative) which inherits from https://st.unicode.org/cldr-apps/v#/root/Duration/616ed987fc730e7c (root, short-other-nominative) -- a different path, BTW.

There are 43 test failures like this one, all units in bs_Cyrl, including duration-nanosecond, graphics-em, ...